src/Common/src/System/Net/SecurityProtocol.cs

```diff
@@ -15,6 +15,8 @@ internal static class SecurityProtocol
         public const SslProtocols DefaultSecurityProtocols =
             SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
 
+        public const SslProtocols SystemDefaultSecurityProtocols = SslProtocols.None;
```

Comment:
>  I recommend changing the CURL implementation to use internal defaults

I'm not entirely sure what this means.  Are you saying that if we see a value of SslProtocols.None, we should just let libcurl do whatever it defaults to, i.e. don't explicitly set it?